index.html

```diff
@@ -432,6 +432,75 @@ <h3>Obtaining Consent</h3>
         </li>
       </ol>
     </section>
+    <section>
+      <h3 id=permissions-integration>Permissions Integration</h3>
+      <p>The Audio Output Devices API is a [=powerful feature=] that is
+      identified by the [=powerful feature/name=] "speaker-selection".</p>
+      <p>It defines the following types and algorithms:</p>
+      <dl>
+        <dt>
+          [=powerful feature/permission descriptor type=]
+        </dt>
+        <dd>
+          <p>
+            A permission covers access to the device given in the associated
+            {{DevicePermissionDescriptor}} descriptor.
+          </p>
+          <p>
+            If the descriptor does not have a {{DevicePermissionDescriptor/deviceId}}, its
+            semantic is that it queries for access to all devices of that class. Thus, if a query
+            for the "speaker-selection" [=powerful feature=] with no
+            {{DevicePermissionDescriptor/deviceId}} returns {{PermissionState/"granted"}}, the
+            client knows that there will never be a permission prompt for an audio output device,
+            and if {{PermissionState/"denied"}} is returned, it knows that no getUserMedia request
+            for an audio output device will succeed.
+          </p>
+          <p>
+            If a permission state is present for access to some, but not all, audio output devices,
+            a query without the {{DevicePermissionDescriptor/deviceId}} will return
+            {{PermissionState/"prompt"}}.
+          </p>
+          <p>
+            Note that a "granted" permission is no guarantee that getUserMedia will succeed. It
```

Comment:
This note seems wrong, and should be removed.

Reply:
It looks like this was added by @alvestrand in https://github.com/w3c/permissions/pull/119 - maybe he can give more context here?

Reply:
This is to cover the case where acquisition of the device fails for other reasons (device does not fit constraints, device initialization failed.....)

Reply:
Cool, happy to delete. Thanks.